agenta-web/src/components/Playground/Views/ParametersCards.tsx

```diff
@@ -72,6 +72,10 @@ export const ModelParameters: React.FC<ModelParametersProps> = ({
     handleParamChange,
 }) => {
     const classes = useStyles()
+    const handleCheckboxChange = (paramName: string, checked: boolean) => {
+        const value = checked ? 1 : 0
```

Comment:
is there a reason we are passing 1/0 based boolean here?

Reply:
Yes, the SDK is designed to interpret 1 as true and 0 as false, and vice versa (true as 1 and 0 as false). But this is an oversight, I'll quickly make the changes now. Thanks for catching that!